### ISSUE TITLE
f1/precision/recall

### ISSUE THREAD

**Issue by LifangD** [xiangrongzeng/copy_re#2]
hi, Dr Zeng, when I use the default setting to train on  webnlg dataset, the f1/precision/recall seems to be 0.00.. and the entity in visualized result  repeated twice. still can't  figure out what's wrong and looking forward to your reply~

**Comment by xiangrongzeng** [comment]
Try learning rate 0.001 and use pre-trained word embedding

**Comment by monojett** [comment]
Thanks, it's exactly the problem of  the learning rate.
